rmf_fleet_adapter/include/rmf_fleet_adapter/agv/RobotUpdateHandle.hpp

```diff
@@ -152,6 +152,16 @@ class RobotUpdateHandle
     /// (warning tier)
     void blocked(std::optional<std::string> text);
 
+    /// Trigger this when you require a replan for your navigation or
+    /// docking request
+    void replan(bool request_replan);
```

Comment:
We don't have to pass in a bool arg for this function. Users can simply call this function to request for a replan. It can return a bool depending on if the replanning succeeded/failed.
`bool replan();`

Reply:
[afef64c](https://github.com/open-rmf/rmf_ros2/pull/235/commits/afef64c0caddacc9db50472b3e3f607d0c174065)